test/specs/scale.category.tests.js

```diff
@@ -156,35 +156,38 @@ describe('Category scale tests', function() {
 		expect(scale.ticks).toEqual(labels);
 	});
 
-	it ('should get the correct label for the index', function() {
-		var scaleID = 'myScale';
-
-		var mockData = {
-			datasets: [{
-				yAxisID: scaleID,
-				data: [10, 5, 0, 25, 78]
-			}],
-			labels: ['tick1', 'tick2', 'tick3', 'tick4', 'tick5']
-		};
-
-		var config = Chart.helpers.clone(Chart.scaleService.getScaleDefaults('category'));
-		var Constructor = Chart.scaleService.getScaleConstructor('category');
-		var scale = new Constructor({
-			ctx: {},
-			options: config,
-			chart: {
-				data: mockData
+	it('should get the correct label for the index', function() {
+		var chart = window.acquireChart({
+			type: 'line',
+			data: {
+				datasets: [{
+					xAxisID: 'xScale0',
+					yAxisID: 'yScale0',
+					data: [10, 5, 0, 25, 78]
+				}],
+				labels: ['tick1', 'tick2', 'tick3', 'tick4', 'tick5']
 			},
-			id: scaleID
+			options: {
+				scales: {
+					xAxes: [{
+						id: 'xScale0',
+						type: 'category',
+						position: 'bottom'
+					}],
+					yAxes: [{
+						id: 'yScale0',
+						type: 'linear'
+					}]
+				}
+			}
 		});
 
-		scale.determineDataLimits();
-		scale.buildTicks();
+		var scale = chart.scales.xScale0;
 
-		expect(scale.getLabelForIndex(1)).toBe('tick2');
+		expect(scale.getLabelForIndex(1, 0)).toBe('tick2');
 	});
 
-	it ('Should get the correct pixel for a value when horizontal', function() {
+	it('Should get the correct pixel for a value when horizontal', function() {
```

Comment:
Mixed style in these, my editor likes this style. I can revert these changes if needed.

Reply:
It should be `it('should ...')` (function call), not sure why ESLint didn't report it.